src/theme/typography.ts

```diff
@@ -72,12 +74,34 @@ const margin: Margin = {
   },
 };
 
+const links: Links = {
+  primary: {
+    color: colors.blueberryClassic,
+    hoverColor: colors.radiantBlueberry,
+    activeColor: colors.dietBlueberry,
+    decoration: 'none',
+  },
+  danger: {
+    color: colors.cherry,
+    hoverColor: colors.strawberry,
+    activeColor: colors.pumpkin,
+    decoration: 'none',
+  },
+  secondary: {
+    color: colors.graphite2B,
+    hoverColor: colors.radiantBlueberry,
```

Comment:
Did you check hover and active colors with @MateCziner? It doesn't seem right that we use blue shades for this

Reply:
Yep, he was so nice as to define all the proper styles. Check here:
https://www.figma.com/file/vnzYfrwIISRYkr2iYE5TcD/Platform-%3A-Atoms?node-id=1972%3A491